docs/userguide/boundaryanalysis/break-rules.md

```diff
@@ -230,110 +241,124 @@ rules](https://github.com/unicode-org/icu/blob/master/icu4c/source/data/brkitr/r
-    numbers are *not* Perl regular expression repeat counts.
-
-2.  Comments are recognized and removed separately from otherwise parsing the
-    rules. They may appear wherever a space would be allowed (and ignored.)
-
-3.  The implicit concatenation of adjacent terms has higher precedence than the
-    '|' operation. "ab|cd" is interpreted as "(ab)|(cd)", not as "a(b|c)d" or
-    "(((ab)|c)d)"
-
-4.  The syntax for UnicodeSet is defined (and parsed) by the UnicodeSet class.
-    It is not repeated here.
-
-5.  For $variables that will be referenced from inside of a UnicodeSet, the
-    definition must consist only of a Unicode Set. For example, when variable $a
-    is used in a rule like \[$a$b$c\], then this definition of $a is ok
-    “$a=\[:Lu:\];” while this one “$a=abcd;” would cause an error when $a was
-    used.
-
-6.  Spaces are allowed nearly anywhere, and are not significant unless escaped.
-    Exceptions to this are noted.
-
-7.  No spaces are allowed within a variable name. The variable name $dictionary
-    is special. If defined, it must be a Unicode Set, the characters of which
-    will trigger the use of word dictionary based boundaries.
-
-8.  A leading '^' on a rule prevents chaining into that rule. It can only match
-    immediately after a preceding boundary, or at the start of text.
-
-9.  {nnn} appearing at the end of a rule is a Rule Status number, not a repeat
-    count as it would be with conventional regular expression syntax.
-
-10. A '/' in a rule specifies a hard break point. If the rule matches, a
-    boundary will be forced at the position of the '/' within the match.
-
-EBNF Syntax used for the RBBI rules syntax description
-
-a?zero or one instance of aa+one or more instances of aa\*zero or more instances
-of aa | beither a or b, but not both'a' "a"the literal string between the quotes
+Here is the syntax for the boundary rules. (The EBNF Syntax is given below.)
+
+| Rule Name | Rule Values | Notes |
+| ---------- | ----------- | ----- |
+| rules | statement+ | |
+| statement | assignment \| rule \| control |
+| control | (`!!forward` \| `!!reverse` \| `!!safe_forward` \| `!!safe_reverse` \| `!!chain`) `;`
+| assignment | variable `=` expr `;` | 5 |
+| rule | `^`? expr (`{`number`}`)? `;` | 8,9 |
+| number | [0-9]+ | 1 |
+| break-point | `/` | 10 |
+| expr | expr-q \| expr `|` expr \| expr expr | 3 |
+| expr-qterm | term `*` \| term `?` \| term `+` |
+| term | rule-char \| unicode-set \| variable \| quoted-sequence \| `(` expr `)` \| break-point |
+| rule-special | *any printing ascii character except letters or numbers* \| white-space |
+| rule-char | *any non-escaped character that is not rule-special* \| `.` \| *any escaped character except* `\p` *or* `\P` |
+| variable | `$` name-start-char name-char* | 7 |
+| name-start-char | `_` \| \p{L} |
+| name-char | name-start-char \| \\p{N} |
+| quoted-sequence | `'''` *(any char except single quote or line terminator or two adjacent single quotes)*+ `'''` |
```

Comment:
The tripled single quotes should just be one single quote, enclosed in backslashes to indicate it's a literal.

Reply:
I've been just using:   `'` (monospace) to indicate a literal, does that work?  See how you like the updated version.